src/parseWithPointers.ts

```diff
@@ -23,8 +23,9 @@ export const parseWithPointers = <T>(value: string, options?: LoadOptions): Yaml
   const parsed: YamlParserResult<T> = {
     ast,
     lineMap,
-    data: {} as T, // fixme: we most likely should have undefined here, but this might be breaking
```

Comment:
Isn't this breaking anymore?

Reply:
It totally is.
I'll release a breaking release.